src/gridInterceptor/interceptorP5.js

```diff
@@ -1,68 +1,68 @@
 funcNames = allData['classitems'].map(function(x) {
-  if (x['overloads']) {
-    tempParam = x['overloads'][0]['params'];
-  } else {
-    tempParam = x['params'];
-  }
-  return {
-    name: x['name'],
-    params: tempParam,
-    class: x['class'],
-    module: x['module'],
-    submodule: x['submodule'],
-  };
+    if (x['overloads']) {
+        tempParam = x['overloads'][0]['params'];
+    } else {
+        tempParam = x['params'];
+    }
+    return {
+        name: x['name'],
+        params: tempParam,
+        class: x['class'],
+        module: x['module'],
+        submodule: x['submodule'],
+    };
 });
 
 funcNames = funcNames.filter(function(x) {
-  var className = x['class'];
-  return (x['name'] && x['params'] && (className === 'p5'));
+    let className = x['class'];
+    return (x['name'] && x['params'] && (className === 'p5'));
 });
-if(document.getElementById('tableOutput-content')) {
-  funcNames.forEach(function(x) {
-    // var document = parent.document;
-    var originalFunc = p5.prototype[x.name];
-    var byID = function(id) {
-      var element = document.getElementById(id);
-      return element;
-    };
-    var details = byID('tableOutput-content-details');
-    var summary = byID('tableOutput-content-summary');
-    var table = byID('tableOutput-content-table');
+if (document.getElementById('tableOutput-content')) {
+    funcNames.forEach(function(x) {
+        // let document = parent.document;
+        let originalFunc = p5.prototype[x.name];
+        let byID = function(id) {
+            let element = document.getElementById(id);
+            return element;
+        };
+        let details = byID('tableOutput-content-details');
+        let summary = byID('tableOutput-content-summary');
+        let table = byID('tableOutput-content-table');
 
-    p5.prototype[x.name] = function() {
-      orgArg = arguments;
+        p5.prototype[x.name] = function() {
+            orgArg = arguments;
 
-      if (frameCount == 0) { // for setup
-        table.innerHTML = '';
-        details.innerHTML = '';
-        summary.innerHTML = '';
-        gridInterceptor.createShadowDOMElement(document);
-        gridInterceptor.setupObject =
-        gridInterceptor.populateObject(x, arguments, gridInterceptor.setupObject, details, false);
-        gridInterceptor.populateObjectDetails(gridInterceptor.setupObject, gridInterceptor.drawObject, summary, details);
-        gridInterceptor.populateTable(details, gridInterceptor.setupObject);
-      } else if (frameCount == 1 || frameCount % 20 == 0) {
-        gridInterceptor.drawObject =
-        gridInterceptor.populateObject(x, arguments, gridInterceptor.drawObject, details, true);
-        gridInterceptor.isCleared = false;
+            if (frameCount == 0) { // for setup
+                table.innerHTML = '';
+                details.innerHTML = '';
+                summary.innerHTML = '';
+                gridInterceptor.createShadowDOMElement(document);
```

Comment:
throwing an error here, might be fixed when line 65 problem is fixed

Reply:
This should be good to go.